app/electron/main.ts

```diff
@@ -515,7 +515,8 @@ function startElecron() {
         query: {
           backendToken: backendToken,
         },
-      });
+      })
+    ).replace(/\\/g, '/');
```

Comment:
Can you add a comment explaining why this is needed?

Reply:
done